docs/root/configuration/overview/extension.rst

```diff
@@ -90,3 +92,20 @@ common subscription statistics, it also provides the following:
   config_reload, Counter, Total number of successful configuration updates
   config_fail, Counter, Total number of failed configuration updates
   config_conflict, Counter, Total number of conflicting applications of configuration updates; this may happen when a new listener cannot reuse a subscribed extension configuration due to an invalid type URL.
+
+
+Listener Filters
+
+For Listener filters, the discovery service configuration is: :ref:`dynamic listener filter
+re-configuration<envoy_v3_api_field_config.listener.v3.ListenerFilter.config_discovery>`.
+Note, the dynamic listener filter config is only supported in TCP listeners.
+
+Listener filter Extension config discovery service has a :ref:`statistics
+<subscription_statistics>` tree rooted at listener.<address>. (or listener.<stat_prefix>. if :ref:`stat_prefix
+<envoy_v3_api_field_config.listener.v3.Listener.stat_prefix>` is non-empty) with the following statistics:
+
+.. csv-table::
+  :header: Name, Type, Description
+  :widths: 1, 1, 2
+
+  extension_config_missing, Counter, Total connections closed due to missing listener filter extension configuration
```

Comment:
It will be better to describe that if the config is missing, the connection will be rejected until the valid config was updated in the above.

Reply:
ACK